prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -285,4 +303,31 @@ public void verify() {
             }
         }
     }
+
+    private void printListOfProfiles() throws Exception {
+        Set<String> featurePackNames = KnownFeaturePacks.getNames();
+        for(String channelNames: featurePackNames){
+            KnownFeaturePack knownFeaturePack =  KnownFeaturePacks.getByName(channelNames);
+            final ChannelPrinter channelPrinter = new ChannelPrinter(console);
+            console.println("---channels----");
+            for (Channel channel : knownFeaturePack.getChannels()) {
+                channelPrinter.print(channel);
+                console.println("---feature packs----");
+                FeaturePackLocation loc = getFpl(knownFeaturePack);
+                console.println(loc.toString());
+
+            }
+        }
+    }
+
+    private FeaturePackLocation getFpl(KnownFeaturePack knownFeaturePack) throws Exception {
+        GalleonProvisioningConfig config = GalleonUtils.loadProvisioningConfig(knownFeaturePack.getGalleonConfiguration());
+        if (config.getFeaturePackDeps().isEmpty()) {
+            throw new ProvisioningException("At least one feature pack location must be specified in the provisioning configuration");
+        }
+        FeaturePackLocation fpl = config.getFeaturePackDeps().iterator().next().getLocation();
+        String[] split = fpl.toString().split(":");
+        return FeaturePackLocationParser.resolveFpl(split[0] + ":" + split[1] + ":");
```

Comment:
You don't need to call to FeaturePackLocationParser.resolveFpl. Also drop the trainiling `:`.